File: counting-sort/counting-sort.py
size = input()
lim = 300
freq = lim * [0]
arr = [int(e) for e in input().split(' ')]

for i in arr:
    freq[i] = freq[i] + 1

for i in range(lim):
    if freq[i] > 0:
        print(i, freq[i])

File: counting-sort/counting-sort.cpp
#include <iostream>
using namespace std;
// namespace std_out = std;
// using std::cout;

int main()
{
    // input
    int range = 300, size, freq[range] = {0};
    cin >> size;
    int arr[size];
    for (int i = 0; i < size; i++)
        cin >> arr[i];

    // count frequency
    for (int i = 0; i < size; i++)
        freq[arr[i]]++;

    for (int i = 0; i < range; i++)
        if (freq[i] > 0)
            cout << i << " " << freq[i] << endl;
}